tailwind.config.ts

```diff
@@ -20,8 +20,17 @@ const config: Config = {
         background: 'var(--background)',
         foreground: 'var(--foreground)',
       },
+      animation: {
+        'move-up-down': 'moveUpDown 4s ease-in-out infinite',
+      },
+      keyframes: {
+        moveUpDown: {
+          '50%': { transform: 'translateY(-50px)' },
+        },
+      },
```

Comment:
I don't see this used anywhere 
why is it here?

Reply:
fixed [d4c46a4](https://github.com/dark-orga/anitube-website/pull/2/commits/d4c46a4fe8ffbf9e2f2a15e41c88e4b31ea311b6)